packages/components/src/button/button.tsx

```diff
@@ -19,6 +19,17 @@ type Props = PressableProps & {
   icon?: React.ReactElement
   iconAfter?: React.ReactElement
   disabled?: boolean
+  fullWidth?: boolean
+  customColors?: {
```

Comment:
I find it preferable for button to accept only single color and derive hover/pressed state by changing the opacity.

Reply:
There should be a pattern, I don't want consumers to be able to pass *any* value.

Reply:
Also I am getting this error in the console:
> React does not recognize the `customColors` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `customcolors` instead. If you accidentally passed it from a parent component, remove it from the DOM element.